.github/workflows/ci-md.yml

```diff
@@ -15,11 +15,14 @@ on:
 
 jobs:
   build-test:
-    runs-on: ubuntu-latest
-
     strategy:
       matrix:
-        version: [net6.0]
+        os: [ windows-latest, ubuntu-latest ]
+        version: [ net462, net6.0, net7.0 ]
+        exclude:
+        - os: ubuntu-latest
+          version: net462
```

Comment:
Just curious... is it important that the matrix for the `md` workflow is identical to the actual CI workflow? Could all of this just be removed?

Reply:
it should be identical, to let CI checks show up as green for those PRs touching only md files.

Reply:
I actually just tried removing the strategy from the workflow on my fork and submitted a PR only modifying a `.md` file and the build workflows were skipped. I'll submit a PR to remove them from the `*-md.yml` file now.